app/index.html

```diff
@@ -44,8 +41,14 @@
             <li>
                 <h2><img src="icons/play.png"/><span>Playback</span></h2>
                 <ul>
-                    <li><a id="changeFrameRate" href="#"> Select frame rate</a></li>
-                    <li><input id="loopCheckbox" type="checkbox" name="options" value="Download"/><span> Loop frames on playback</span></li>
+                    <li>
+                      <label id="label-fr-change" for="input-fr-change">Select frame rate</label>
+                      <input id="input-fr-change" type="number" value="15">
+                    </li>
+                    <li>
+                      <input id="loopCheckbox" type="checkbox" name="options" value="Download">
+                      <label for="loopCheckbox"><span>Loop frames on playback</span></label>
```

Comment:
I gave the checkbox a `<label>` so users are not forced to click the check, but now can click the text and it will work (also a better UX).